distribution/scripts/netty/start-netty.sh

```diff
@@ -74,5 +75,10 @@ fi
 mkdir -p logs
 
 echo "Starting Netty"
-nohup java -Xms${heap_size} -Xmx${heap_size} -XX:+PrintGC -XX:+PrintGCDetails -XX:+PrintGCDateStamps -Xloggc:$gc_log_file \
-    -jar $jar_name $netty_service_flags >netty.out 2>&1
+if [ $enable_ssl = true ]; then
+    nohup java -Xms${heap_size} -Xmx${heap_size} -XX:+PrintGC -XX:+PrintGCDetails -XX:+PrintGCDateStamps -Xloggc:$gc_log_file \
+    -jar $jar_name --ssl $enable_ssl --key-store-file /usr/lib/ballerina/distributions/ballerina-slp7/bre/security/ballerinaKeystore.p12 --key-store-password ballerina >netty.out 2>&1
```

Comment:
The ballerina version also should be parameterized here.
```suggestion
    -jar $jar_name --ssl $enable_ssl --key-store-file /usr/lib/ballerina/distributions/ballerina-slp7/bre/security/ballerinaKeystore.p12 --key-store-password ballerina >netty.out 2>&1
```